docs/usage/step3.ipynb

```diff
@@ -89,7 +89,6 @@
     "    expression=model.expression.doit(),\n",
     "    parameters=model.parameter_defaults,\n",
     "    backend=\"jax\",\n",
-    "    max_complexity=100,\n",
```

Comment:
@Leongrim this may be of importance to your benchmark comparison. (See also #103)